File: dota-zone/dotazone/src/main/java/br/com/dotazone/view/activity/BaseActivity.java
package br.com.dotazone.view.activity;

import android.os.Bundle;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentActivity;
import androidx.fragment.app.FragmentTransaction;
import android.view.Window;

import br.com.dotazone.R;
import br.com.dotazone.model.listeners.ErrorListener;
import br.com.dotazone.model.listeners.Initializable;

abstract public class BaseActivity extends FragmentActivity implements Initializable, ErrorListener {

    @Override
    protected void onCreate(Bundle arg0) {
        super.onCreate(arg0);
        this.requestWindowFeature(Window.FEATURE_NO_TITLE);
    }

    /**
     * Start fragment in content activity.
     *
     * @param fragment
     */
    protected void startFragment(Fragment fragment) {

        FragmentTransaction fragmentTransaction = getSupportFragmentManager().beginTransaction();
        fragmentTransaction.replace(R.id.main_activity_content, fragment);
        fragmentTransaction.commit();
    }
}

File: dota-zone/app/src/main/java/com/br/dotazone/view/activity/BaseActivity.kt
package com.br.dotazone.view.activity

import ErrorListener
import android.os.Bundle
import android.view.Window
import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentActivity
import com.br.dotazone.R
import com.br.dotazone.model.listeners.Initializable


abstract class BaseActivity : FragmentActivity(), Initializable, ErrorListener {
	override fun onCreate(arg0: Bundle?) {
		super.onCreate(arg0)
		requestWindowFeature(Window.FEATURE_NO_TITLE)
	}

	/**
	 * Start fragment in content activity.
	 *
	 * @param fragment
	 */
	protected fun startFragment(fragment: Fragment?) {
		val fragmentTransaction = supportFragmentManager.beginTransaction()
		fragmentTransaction.replace(R.id.main_activity_content, fragment!!)
		fragmentTransaction.commit()
	}
}
